<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>819</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Thermal texturing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>730</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="ok_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>441</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Calibration setup</string>
      </property>
      <widget class="QRadioButton" name="radio_calculate_matrices">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculate matrices</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>431</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="calculate_matrices_opt_layout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout8">
          <item>
           <widget class="QPushButton" name="tv_calibration_files_button">
            <property name="text">
             <string>Select TV calibration files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tv_calibration_files_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout9">
          <item>
           <widget class="QPushButton" name="rgb_calibration_files_button">
            <property name="text">
             <string>Select RGB calibration files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rgb_calibration_files_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Corresponding TV and RGB photos:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="rgb_tv_table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="rowCount">
           <number>1</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>RGB photo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TV photo</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="calibration_file_options_layout_3">
          <item>
           <widget class="QCheckBox" name="save_matrices_checkbox">
            <property name="text">
             <string>Save matrices to following file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="save_matrices_file_edit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="radio_use_file_matrices">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>261</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Use calibration matrices from file</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Select thermal and rgb photos matching</string>
      </property>
      <widget class="QRadioButton" name="radio_by_name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>By name</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_by_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>By distance</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>85</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="by_name_options_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="rgb_index_label">
            <property name="text">
             <string>TV prefix:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rgb_prefix_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="rgb_prefix_label">
            <property name="text">
             <string>TV index:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rgb_index_edit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="tv_index_label">
            <property name="text">
             <string>RGB prefix:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tv_prefix_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="tv_prefix_label">
            <property name="text">
             <string>RGB index:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tv_index_edit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>650</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="calibration_file_options_layout">
        <item>
         <widget class="QPushButton" name="select_calibration_file_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="select_calibration_file_edit"/>
        </item>
       </layout>
      </widget>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ok_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok_pressed()</slot>
 </slots>
</ui>
